app/views/posts/show.html.erb

```diff
@@ -1,7 +1,11 @@
 <section class="show_user_posts">
     <div class= "post-details post-border mt-2">
             <h2>Post #<%= @post.id%> by: <%= @post.title%></h2>
-            <div class="comment-num">Comments: 10, Likes: 10k</div>
+            <div class="comment-num">Comments: <%= @post.comments.count%>, 
+             <%= link_to "like", user_post_likes_path(@user.id, @post.id), method: :post, class: "mx-auto justify-self-center"%>: 
+             <%= @post.likes.count%>
+             
+             </div>
         <p>
             lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
             lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
```

Comment:
- [ ] Instead of displaying the comments and likes manually like that, you can add the add button or icon for liking the comment so that a user should click on it and see the likes.
![form5](https://user-images.githubusercontent.com/55635977/141620193-0a7c521f-d553-4bdb-b6fd-9335112dc6e6.png)

 - [ ] I would recommend adding the like icon and changing the button to dislike If a user clicks on like because a user needs to like once.
 
![form6](https://user-images.githubusercontent.com/55635977/141621444-36c8f8a7-ea97-4d36-8a13-3d0e92b5886f.png)

Reply:
- [ ]  Add likes dynamically

![Screenshot from 2021-11-13 15-13-11](https://user-images.githubusercontent.com/65068771/141645109-c0baef7f-be42-4fb8-8bd1-b100fd2a423a.png)

- [ ]  Add dislike event and an icon

![Screenshot from 2021-11-13 15-11-27](https://user-images.githubusercontent.com/65068771/141645084-e1a7cb91-bf17-437b-8ad2-33f76d9cf14f.png)